samples/docdb-scaling/scaleupdown.py

```diff
@@ -0,0 +1,204 @@
+import boto3
+import json
```

Comment:
I don't see json and os modules being used, remove if that's the case.

Reply:
removed